### ISSUE TITLE
Doc Clarification: do I need a toolchain specific for RI5CY?

### ISSUE THREAD

**Issue by tcal-x** [QuickLogic-Corp/arnold-sdk#1]
I have a few generic riscv toolchains installed (stuff like `riscv64-unknown-elf-gcc`).   Do I need a toolchain specific for RI5CY?

**Comment by gmartin102** [comment]
Yes you need the riscv32 variants.  from a clean cloned arnold-sdk, the command
bash install.bash ++riscv-toolchain
will install and build the proper toolchain and install it into /opt/riscv
